app/src/main/java/com/mobile/tandil/javabaseproject/mvp/model/ParkingModel.java

```diff
@@ -2,4 +2,17 @@
 
 import com.mobile.tandil.javabaseproject.mvp.contract.ParkingContract;
 
-public class ParkingModel implements ParkingContract.Model {}
+public class ParkingModel implements ParkingContract.Model {
+
+    private int parkingAvailable = 0;
```

Comment:
by default, this can be set to a -1 or any other impossible value, to show that it is uninitialized

Reply:
fixed